legal-api/src/legal_api/core/meta/filing.py

```diff
@@ -493,6 +493,10 @@ def alter_outputs(filing: FilingStorage, outputs: set):
                     outputs.remove('certifiedMemorandum')
                 if filing.filing_json['filing']['alteration'].get('rulesInResolution') is True:
                     outputs.remove('certifiedRules')
+        elif filing.filing_type == 'dissolution' and filing.filing_sub_type == 'administrative' and \
+                filing.submitter_roles == 'staff':
```

Comment:
I don't think the check for `filing.submitter_roles == 'staff'` is required?

Reply:
I think it's required - this is task description ` When filing an Admin Dissolution, which is a Staff filing, stop generating a Certificate of Dissolution.`

Reply:
The goal of this ticket is to suppress certificate of dissolution for staff and general users for admin dissolution filings right?  To identify a admin dissolution, you only need to use filing type and filing sub-type.

Reply:
I did